magento/devdocs#7116: Feedback on page: /guides/v2.3/install-gde/system-requirements-tech.html (#7121)
magento/devdocs: src/guides/v2.3/install-gde/system-requirements-tech.md

  - Setup
---

{:.bs-callout-info}

> All of the requirements described in this document are true to the latest patch release of Magento 2.3.

Reviewer: ```suggestion
All of the requirements described here apply to the latest patch release of Magento 2.3.
```

Author: **done**
